### PR TITLE
Small doc change to Updating page

### PR DESCRIPTION
**Pull request by DrrnDyl**
This includes information about upgrading to ver 0.12.

### CODE REVIEW

**Comment by daviddarnes** [comment]
Awesome! Thanks 😄

**Comment by daviddarnes** [comment]
I'll get onto this as soon as I can

**Comment by DrrnDyl** [comment]
No problem. I know you're busy.

**Comment by TheBrenny** [comment]
From what I see, everything looks alright. Might need to tweak the pictures for uniformity across the pages. But that's just me being picky.

File: managing-content/categories.md [hunk 1]
```diff
@@ -5,4 +5,17 @@ title: Categories
 
 # Categories
 
-*We are working to get these pages completed, if you have any questions please post on the [forums](http://forums.anchorcms.com/)*
+Before you can assign categories to your blog you'll need to manually add them:
```
Comment (daviddarnes) [inline managing-content/categories.md]: Can this be _"Before you can assign categories to your blog you'll need to create them"_, sounds more positive 😄

File: managing-content/categories.md [hunk 2]
```diff
@@ -5,4 +5,17 @@ title: Categories
...
+
+<span class="screenshot">![Anchor’s categories screen ](/images/categories-list.png)</span>
+
+Hit the 'create a new category' button on this screen and you'll be taken to the screen below:
+
+<span class="screenshot">![Anchor’s new categories screen](/images/categories-new.png)</span>
```
Comment (daviddarnes) [inline managing-content/categories.md]: Same as mentioned on line 10

**Comment by daviddarnes** [comment]
@TheBrenny agreed, I made a couple of comments just for help with consistency. I really need to set this repo up so PRs get merged straight into the live site, just to make things quicker

**Comment by DrrnDyl** [comment]
@daviddarnes @daviddarnes Ok, made the requested changes although I couldn't do anything about the screenshots as I don't own a Mac.

**Comment by daviddarnes** [comment]
@DrrnDyl don't worry about the screenshots being a mac or not 😄

**Pull request merged**